actions/java-gradle-release/action.yaml

```diff
@@ -2,18 +2,31 @@ name: "Release Java artifacts"
 description: "Release Java Gradle artifacts on Github"
```

Comment:
Is this supposed to be part of this PR? According to your description this is not related to https://github.com/bakdata/ci-templates/issues/25

Reply:
right..that is due to the fact that in order to consider the PR related to using default for changelog i needed to test it. I need a proper repo to test it. ANd while testign i reliased that i cannot just test here because the azure pipeline is used..that's how i ended modifying the graddle action as well

Reply:
i could either rename the PR or separate the graddle related changes